cockpit_api/src/main/java/com/cockpit/api/model/dao/Impediment.java

```diff
@@ -14,11 +14,19 @@
     private String name;
     private String description;
 
-    @ManyToOne(cascade = CascadeType.MERGE)
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
With (cascade = CascadeType.ALL), it means that when we create/update/delete an impediment, the concerned sprint will be created/updated/deleted, too, is that what we want ?
For me the (cascade=CascadeType.ALL) could be added in Sprint DAO for impediments instead of in Impediment DAO for sprint